### ISSUE TITLE
Please add keyboard maneuver support

### ISSUE THREAD

**Issue by makedir** [dumbeau/AutoHotPie#129]
Please add a global way to maneuver through the pie menus with keyboard and execute.

Example:

- button A opens pie menu
- arrow keys let you maneuver through the pie menus where left arrow goes to the left side, right goes to the right side, up to top, down to bottom. and while youre inside the pie menu right goes one up clock wise, left goes down counter clock wise
- hitting enter executes the pie selection

**Comment by dumbeau** [comment]
This already exists in an indirect way, the 'slice hotkey' can be set so that your pie menus function this way.  However there are some issues with slice hotkeys in some situations that I'm working on a patch for.  So this will work better when that patch comes.

![image](https://github.com/dumbeau/AutoHotPie/assets/36665762/bf04de99-da91-4933-a012-0db52c2db27e)

The benefit to this working this way is that you can set 's' to a pie menu, and if you have an 8 slice menu you can use 'q w e a d z x c' (all the key around s) as your navigation keys.  An option could be created to set this up automatically though as opposed to having to set these one-by-one.

**Comment by makedir** [comment]
@dumbeau No, it does not exist. The slice hotkeyS make zero sense, because it is a key for ONE slice ONLY and you need infinite keys for each slice one.

Please add what I suggested here, it is important. There is a fundamental difference between what I asked.

I want maneuver keys so you have just 2 keys which let you circle around the pie in clock and counter clock wise way with just TWO keys.

This is important so you can bind them for example on Xbox controller.

**Comment by dumbeau** [comment]
Ah my bad, hasty assumption on my part. I was responding in the middle of a work-day and didn't take the time to fully understand the request.  I understand now.

Your initial comment is clear enough, and the function seems straight forward.  Would you want the navigation keys to be re-assignable?  I think committing to the arrow keys and enter seems reasonable.

**Comment by makedir** [comment]
I think best would be:

- enable this as optional on/off
- assign 2 or 4 maneuver keys, for example arrow keys, pressing them would enter the pie menue and if youre on the shell of the pie menu pressing them would then go clock or counter clock wise
- assign an enter key to activate the selected pie slice, for example enter
- could be nice if you could change these keys too and not hard coded

**Comment by dumbeau** [comment]
Copy that, I'll add assignable navigation functions and a simple enable checkbox.  That makees sense with the current design pattern of having all inputs be re-assignable also.  This can be in the next version.

**Comment by makedir** [comment]
Thank you very much. This will help to use the pie menu with a xbox controller where you assign the arrow and enter keys to.
